pkg/cmd/cli/describe/describer.go

```diff
@@ -78,7 +78,7 @@ func (d *BuildDescriber) DescribeParameters(p buildapi.BuildParameters, out *tab
 			formatString(out, "BaseImage", p.Strategy.DockerStrategy.BaseImage)
 		}
 	case buildapi.STIBuildStrategyType:
-		formatString(out, "Builder Image", p.Strategy.STIStrategy.Image)
+		formatString(out, "Builder Image", p.Strategy.STIStrategy.From.Name)
```

Comment:
output the namespace if set like we do elsewhere:
		if p.Output.To.Namespace != "" {
			formatString(out, "Output to", fmt.Sprintf("%s/%s", p.Output.To.Namespace, p.Output.To.Name))
		} else {
			formatString(out, "Output to", p.Output.To.Name)
		}